libwnbd/wnbd_ioctl.cpp

```diff
@@ -95,10 +99,17 @@ DWORD WnbdOpenAdapterEx(PHANDLE Handle, PDEVINST CMDeviceInstance)
         *CMDeviceInstance = DevInfoData.DevInst;
     }
     else {
-        LogError(
-            "Could not oped WNBD device. Please make sure that the "
-            "driver is installed. Error: %d. Error message: %s",
-            ErrorCode, win32_strerror(ErrorCode).c_str());
+        if (ErrorCode == ERROR_ACCESS_DENIED) {
+            LogError(
+                "Could not open WNBD adapter device. Access denied, try "
```

Comment:
Should the message be shortened to `Access denied, using an elevated command prompt.`?

Reply:
It's not really the same thing, "Access denied, using an elevated command prompt." suggests that we're automatically using an elevated prompt as a fallback, which is confusing.